zeal.js

```diff
@@ -479,25 +194,24 @@ $("#read-button").on('click', function() {
     let eepromr = new FileReader();
     eepromr.addEventListener('load', function(e) {
         let binary = e.target.result;
-        eeprom.loadFile(binary);
+        zealcom.eeprom.loadFile(binary);
         $("#eepromready").addClass("ready");
     });
     if (typeof file !== "undefined") {
         eepromr.readAsBinaryString(file);
     }
 });
 
-
 $("#screen").on("keydown", function(e) {
-    const handled = keyboard.key_pressed(e.keyCode);
+    const handled = zealcom.keyboard.key_pressed(e.keyCode);
 
     if (handled) {
         e.preventDefault();
     }
 });
 
 $("#screen").on("keyup", function(e) {
-    const handled = keyboard.key_released(e.keyCode);
+    const handled = zealcom.keyboard.key_released(e.keyCode);
```

Comment:
Similarly, I wonder if it would be more meaningful to have a public API for this:
```
KeyboardKeyPressed(e.keyCode);
KeyboardKeyRelease(e.keyCode);
```
given by the `Machine` object

Reply:
Added.